java-design-patterns/src/main/java/org/example/Main.java

```diff
@@ -1,6 +1,7 @@
 package org.example;
 
 import org.example.Singleton.SingletonCache;
+import org.example.factoryMethod.*;
 
 public class Main {
```

Comment:
better to name the class same as functional behaviour. this is a factory class may be something like `CalculatorFactory`